From 46569984a65de91d9b2e570fea54d4c8a443a4c7 Mon Sep 17 00:00:00 2001
From: Jon Medhurst <tixy@linaro.org>
Date: Mon, 5 Nov 2012 17:30:50 +0000
Subject: [PATCH] configs: android: Don't disable CONFIG_THUMB2_KERNEL

This was originally done to workaround a toolchain bug which
seems to have now been resolved. See
https://bugs.launchpad.net/linaro-landing-team-arm/+bug/973466

Signed-off-by: Jon Medhurst <tixy@linaro.org>
---
 linaro/configs/android.conf | 1 -
 1 file changed, 1 deletion(-)

diff --git a/linaro/configs/android.conf b/linaro/configs/android.conf
index 9984a92..1518036 100644
--- a/linaro/configs/android.conf
+++ b/linaro/configs/android.conf
@@ -1,4 +1,3 @@
-# CONFIG_THUMB2_KERNEL is not set
 CONFIG_IPV6=y
 # CONFIG_IPV6_SIT is not set
 CONFIG_PANIC_TIMEOUT=0
-- 
1.8.0

